schema.graphql

```diff
@@ -152,3 +154,18 @@ type Transaction @entity {
   earnings: BigDecimal
   gysrSpent: BigDecimal
 }
+
+type Funding @entity {
+  # geyser id, created at timestamp
+  id: ID!
+
+  geyser: Geyser!
+  token: Token!
+
+  createdTimestamp: BigInt!
+  start: BigInt!
+  end: BigInt!
+
+  sharesPerSecond: BigDecimal!
+  shares: BigDecimal!
```

Comment:
you could still store an original amount value here to make event matching easier later

Reply:
(though in v2, we will emit shares in the funding event, so matching will be easy anyways)